src/elements/content-preview/ContentPreview.js

```diff
@@ -108,6 +108,7 @@ type Props = {
         [name: string]: TargetingApi,
     },
     previewLibraryVersion: string,
+    renderAvatar?: Function,
```

Comment:
We generally want to avoid exposing more props on our top-level Elements. Can we pass this new value in via the existing `activitySidebarProps` object, instead?

Reply:
That sounds like a great idea, thanks. I wasn't sure what those properties are doing. Also that's my first PR to box-ui-elements and I'm still not familiar with how things are being done here. Let me refactor it...